project-backend/Providers/JobProvider/JobProvider.cs

```diff
@@ -33,16 +35,24 @@ public JobDAO AddJob(string name, string description, int userId)
 
         public JobDAO GetJobById(int jobId)
         {
-            JobDAO requiredJob;
             try
             {
-                requiredJob = _dbContext.Jobs.AsQueryable().Where(job => job.Id == jobId).First();
+                JobDAO requiredJob = _dbContext.Jobs.Include(job => job.Images).Where(job => job.Id == jobId).First();
+                return requiredJob;
             }
             catch (InvalidOperationException)
             {
                 throw new ResourceNotFoundException("Job with that id doesn't exist!");
             }
-            return requiredJob;
+
+        }
+
+
+        public IQueryable<BidDAO> GetJobBids(int jobId)
```

Comment:
Shouldn't this be in the bid provider?

Reply:
True, but they are also stored separately, could use more input on this.

Reply:
Just an opinion, but maybe putting this method in the bid provider would make it easier for someone to discover it because looking at the provider for a specific entity should show you all the ways to interact with that entity, i.e. you shouldn't have to use a bid provider to add bids to a job, but use a job provider to get the bids of a job

Reply:
Moved it to the BidProvider.